File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[]arr = new int[2*n];
            int j=0;
            for (int i=0;i<nums.length-1;i+=2){
                arr[i]=nums[j];
                arr[i+1]=nums[n+j];
                j++;
            }
            return arr;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.kt
import java.util.stream.IntStream


class Solution {
fun shuffle(nums: IntArray, n: Int): IntArray {
    val result = IntArray(nums.size)

    for (i in 0 until n) {
        result[2*i] = nums[i]
        result[2*i+1] = nums[i+n]
    }
    return result
}
}